recipe.yml

```diff
@@ -18,7 +18,7 @@ modules:
   - apt upgrade -y
   - apt clean
   - apt-mark hold snapd gnome-software-plugin-snap
-  - apt install minisign
+  - apt -y install unsquashfs-tools minisign
```

Comment:
should be `[squashfs-tools](open:https://packages.vanillaos.org/package/squashfs-tools)`